eZ/Publish/Core/MVC/Symfony/Templating/RenderStrategy.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\Templating;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\SPI\MVC\Templating\RenderStrategy as RenderStrategyInterface;
+
+class RenderStrategy implements RenderStrategyInterface
+{
+    /** @var \eZ\Publish\SPI\MVC\Templating\RenderStrategy[] */
+    private $strategies;
+
+    public function __construct(iterable $strategies)
+    {
+        $this->strategies = $strategies;
+    }
+
+    public function supports(ValueObject $valueObject): bool
```

Comment:
I don't know that we want to stick strictly to terminology but for me, it looks more like Specification than strategy ;)

Reply:
This is still `Stategy`, just with an IoC predicate